python/lsst/ts/hexrotcomm/structs.py

```diff
@@ -45,6 +45,7 @@ class Command(ctypes.Structure):
         ("param5", ctypes.c_double),
         ("param6", ctypes.c_double),
     ]
+    COMMANDER = 1  # value of the commander field for a CSC
```

Comment:
2 instead of 1:
https://github.com/lsst-ts/ts_hexrotpxicom/blob/develop/include/interface/commandStructure.h#L50-L59

```
typedef enum {
    // Graphical user interface
    Commander_GUI = 1,
    // Commandable SAL component
    Commander_CSC = 2,
    // Controller internal use. This is used in the hexapod controller code to
    // hack the Simulink model. This value should be removed after fixing the
    // hexapod Simulink model.
    Commander_Self = 3,
} Commander;
```

Reply:
Good catch. Fixed.